dll/win32/shell32/lang/ro-RO.rc

```diff
@@ -588,55 +588,55 @@ CAPTION "Închidere ReactOS"
 FONT 8, "MS Shell Dlg"
 BEGIN
     ICON 8240, -1, 6, 6, 21, 20, SS_REALSIZECONTROL | WS_GROUP
-    LTEXT "Ce doriți să se întâmple?", -1, 39, 7, 167, 10
+    LTEXT "Ce acţiune va face computerul?", -1, 39, 7, 167, 10
     COMBOBOX 8224, 39, 20, 165, 200, CBS_DROPDOWNLIST | WS_VSCROLL
-    LTEXT "Aceasta va menține sesiunea de lucru, păstrând calculatorul pe consum redus, cu toate datele în memorie. Calculatorul își va reveni când veți apăsa o tastă sau veți acționa dispozitivul de indicare.", 8225, 39, 40, 167, 37
+    LTEXT "Menţine sesiunea deschisă, lăsând computerul să funcţioneze la un nivel de energie scăzut, datele rămânând în memorie. Computerul se reactivează când se apasă o tastă sau când se acţionează mausul.", 8225, 39, 40, 167, 37
     DEFPUSHBUTTON "OK", 1, 7, 82, 60, 14, WS_GROUP
     PUSHBUTTON "Revocare", IDCANCEL, 75, 82, 60, 14
-    PUSHBUTTON "&Manual…", IDHELP, 144, 82, 60, 14
+    PUSHBUTTON "Aj&utor", IDHELP, 144, 82, 60, 14
 END
 
 IDD_FORMAT_DRIVE DIALOGEX 50, 50, 184, 218
 STYLE DS_SHELLFONT | DS_MODALFRAME | DS_CONTEXTHELP | WS_POPUPWINDOW | WS_VISIBLE | WS_CAPTION
-CAPTION "Formatare unitate de stocare"
+CAPTION "Format"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    DEFPUSHBUTTON "&Formatează", IDOK, 53, 198, 60, 14
-    PUSHBUTTON "Î&nchide", IDCANCEL, 118, 198, 60, 14
+    DEFPUSHBUTTON "&Start", IDOK, 53, 198, 60, 14
+    PUSHBUTTON "Î&nchidere", IDCANCEL, 118, 198, 60, 14
```

Comment:
It is really "I&nchidere" here? Not "Revocare"? A simple "Yes" is enough. I believe you without a pic here if you say so.

Reply:
I will look on it tomorrow.

Reply:
> It is really "I&nchidere" here? Not "Revocare"? A simple "Yes" is enough. I believe you without a pic here if you say so.

Yes

Sorry for my late reply.

Reply:
Ok, resolved then. Thx!